ledger/ledger-api-common/src/main/scala/com/digitalasset/platform/server/api/validation/ErrorFactories.scala

```diff
@@ -93,7 +93,7 @@ class ErrorFactories private (errorCodesVersionSwitcher: ErrorCodesVersionSwitch
             .build()
         },
         v2 = LedgerApiErrors.InternalError
-          .CommandTrackerInternalError(
+          .QueueBufferError(
             "Missing status in completion response.",
```

Comment:
You're right, I oversaw that caller. This error constructor doesn't have anything specific internally. Renamed it to `Generic`, as I believe it would be quite handy when someone just needs a thin wrapper of an internal error.